tests/test_data/configs/disabled_override_plugin_config.toml

```diff
@@ -0,0 +1,2 @@
+[connections.test]
```

Comment:
Should we add the test for not-enabled plugin as well? That is:
```
[cli.plugin.something]
enabled = false
```

Reply:
This is that case `not in config` == disabled